1.16_combat-6/src/main/java/io/github/axolotlclient/modules/hud/gui/hud/KeystrokeHud.java

```diff
@@ -155,25 +163,25 @@ public Keystroke createFromString(Rectangle bounds, DrawPosition offset, KeyBind
 	}
 
 	@Override
-	public void render(MatrixStack graphics, float delta) {
-		graphics.push();
+	public void render(AxoRenderContext graphics, float delta) {
+		graphics.br$pushMatrix();
 		scale(graphics);
 		renderComponent(graphics, delta);
-		graphics.pop();
+		graphics.br$popMatrix();
 	}
 
 	@Override
-	public void renderComponent(MatrixStack graphics, float delta) {
+	public void renderComponent(AxoRenderContext graphics, float delta) {
 		if (keystrokes == null) {
 			setKeystrokes();
 		}
 		for (Keystroke stroke : keystrokes) {
-			stroke.render(graphics);
+			stroke.render((MatrixStack) graphics);
```

Comment:
I'm not too happy about these casts, ideally it should use the render context impl all the way down